critiquebrainz/frontend/views/review.py

```diff
@@ -33,20 +33,20 @@ def browse():
         return redirect(url_for('.browse'))
     limit = 3 * 9  # 9 rows
     offset = (page - 1) * limit
-    reviews, count = Review.list(sort='created', limit=limit, offset=offset)
-
-    if not reviews:
-        if page - 1 > count / limit:
+    reviews_release_group, count_release_group = Review.list(sort='created', entity_type='release_group', limit=limit, offset=offset)
+    reviews_event, count_event = Review.list(sort='created', entity_type='event', limit=limit, offset=offset)
```

Comment:
I think it will be better if you load only one type of reviews on a page. For example, have a query parameter `entity_type` and do filtering based on it.